### ISSUE TITLE
bibtex is paranoid

### ISSUE THREAD

**Issue by DeusProx** [andrewrynhard/atom-latex-plus#56]
Latex-Plus couldn't build because bibtex can't write into the .latex folder.
Running the command from the terminal gives the following:

```
$ bibtex .latex/seminar.aux 

bibtex: Not writing to .latex/seminar.blg (openout_any = p).
I couldn't open file name `.latex/seminar.blg'
```

As much as I like hided directories for such files the default paranoid behaviour prevents this. Making everyone change their system wide configs for latex-plus wouldn't be great. I couldn't find any flags to just change this behaviour just for the current execution.

**Comment by DeusProx** [comment]
This is probably a bit related to #49

**Comment by DeusProx** [comment]
The following works from the terminal but I don't know the responsible line in the source code to see of that is possible or if this somehow can be activated by an environment variable just for latex-plus/atom.

```
$ openout_any=a bibtex .latex/seminar.aux 
This is BibTeX, Version 0.99d (TeX Live 2015/Debian)
The top-level auxiliary file: .latex/seminar.aux
The style file: plain.bst
Database file #1: seminar.bib
```

**Comment by andrewrynhard** [comment]
This seems to be a security feature : http://tex.stackexchange.com/questions/223870/bibtex-error-not-writing-to-book-blg-openout-any-p-mactex-2014. Is this happening when you try to compile? As a workaround you can `export openout_any=a` and then `atom ~/path/to/project` and latex-plus should pick up this [environment variable](https://github.com/andrewrynhard/atom-latex-plus/blob/master/lib/environment.js#L29)

**Comment by DeusProx** [comment]
Yeah. My bibfile doesn't change that often so I can just run it myself from terminal. Still the default setting is paranoid and I hope this package will support the default in future.
I took the following from my texmf.cnf:

```
% Allow TeX \openin, \openout, or \input on filenames starting with `.'
% (e.g., .rhosts) or outside the current tree (e.g., /etc/passwd)?
% a (any)        : any file can be opened.
% r (restricted) : disallow opening "dotfiles".
% p (paranoid)   : as `r' and disallow going to parent directories, and
%                  restrict absolute paths to be under $TEXMFOUTPUT.
output_any=p
....
```

It would just be good if we would remove the leading dot from our output directory. I too would like it much more with the leading dot but in respect to security it should change this way.

**Comment by DeusProx** [comment]
Closed by #71

**Comment by DeusProx** [comment]
Closed by #71
